Sources/HipMobileUI.iOS/HipMobileUI.iOS.csproj

```diff
@@ -488,27 +488,21 @@
   <ItemGroup>
```

Comment:
You can leave this file out: These changes are always automatically done when you compile the project